content/tutorials/reactjs-realtime-commenting.textile

```diff
@@ -22,140 +22,145 @@ tags:
 
-```[sh]
-npm install -g create-react-app
-```
+bc..
+npm install -g create-react-app@3.4.1
+div.
 
 Once installed, we can use it to create a new React app. We'll call it @reactjs-realtime-commenting@:
 
-```[sh]
+bc..
 create-react-app reactjs-realtime-commenting
-```
+div.
 
 The new React app is ready. We can start it by running:
 
-```[sh]
+bc..
 cd reactjs-realtime-commenting
 npm start
-```
+div.
 
 This builds the static site, runs a web server and opens a browser. If your browser does not open, navigate to "http://localhost:3000":http://localhost:3000.
 
-"See this step in Github":https://github.com/ably/tutorials/tree/reactjs-commenting-step2
+"See this step in Github":https://github.com/marsielko/tutorials/tree/c4815ffbb58a97be4d5848c37ab4acd11b63b21d
 
 h2. Step 3 - Delete Unused Files
 
-Before we move on to building upon this app, let’s quickly delete some files we won’t be needing. Run the command below to do so:
+Before we move on to building upon this app, let’s quickly delete some files we won’t be needing: @src/App.css@, @src/App.test.js@ and @src/logo.svg@. Run the command below to do so:
 
-```[sh]
-rm src/App.css src/App.test.js src/index.css src/logo.svg
-```
+bc..
+rm src/App.css src/App.test.js src/logo.svg
+div.
 
-"See this step in Github":https://github.com/ably/tutorials/tree/reactjs-commenting-step3
+"See this step in Github":https://github.com/marsielko/tutorials/tree/9db6c81a6dc24d4d8ee7008ce895e67e4eed3ef5
 
 h2. Step 4 - Create Component Folder
 
 Now, we’ll create a new folder within the @src@  folder called @components@. This folder will contain all our React components.
 
-```[sh]
+bc..
 mkdir src/components
-```
+div.
 
 Once that is created, we need to move @App.js@ into the @components@ folder:
 
-```[sh]
+bc..
 mv src/App.js src/components
-```
+div.
 
-"See this step in Github":https://github.com/ably/tutorials/tree/reactjs-commenting-step4
+"See this step in Github":https://github.com/marsielko/tutorials/tree/315e379ea668e44aa7348a6efea76d979a2eab7c
 
 h2. Step 5 - Remove References To Unused Files
 
-Since we have deleted and moved some files which are still being referenced in @src/index.js@ and @src/App.js@, we need to update these files to not reference the deleted and moved files. Firstly, update @src/index.js@ as below:
+Since we have deleted and moved some files which are still being referenced in @src/index.js@ and @src/components/App.js@, we need to update these files to not reference the deleted and moved files. Firstly, update @src/index.js@ as below:
 
-```[javascript]
+bc..
```

Comment:
again, i see you have removed `javascript` language. i think this might affect how language switcher works.
(and again I need to pull the repo and check this for myself)

Reply:
Same as above, I just replaced markdown syntax that was used with a textile syntax, that should be used based on file type.